scss/detail.scss

```diff
@@ -1,74 +1,84 @@
-.container {
+.detail {
```

Comment:
Something very important you don't need move sass classes when add new parent element, this a specificity to selector and we don't want it

so use classes in this form 

.detail {
  ...
}

.card {
  ...
}

card don't need be inside .detail

Reply:
Thanks Cristian I will keep this in mind